packages/sample-data/src/core2/testClozeSpec.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  ClozeTaskContent,
+  MemoryTaskSpec,
+  TaskContentType,
+  TaskSpecType,
+} from "@withorbit/core2";
+
+export const testClozeSpec: MemoryTaskSpec<ClozeTaskContent> = {
```

Comment:
_PS_: I moved this out of `sample-data/src/core2/testQASpec.ts` to match the layout of the sample core1 types (i.e. `sample-data/src/testClozePrompt.ts`).